src/App.js

```diff
@@ -11,35 +17,152 @@ class App extends React.Component {
       cardDeck: makeShuffledDeck(),
       // currCards holds the cards from the current round
       currCards: [],
+      cardsLeft: 52,
+      gameStarted: false,
+
+      playerOneCurrWinner: false,
+      playerOneScore: 0,
+      playerTwoScore: 0,
     };
   }
 
+  playerOneIsHigher = (list) => {
+    const suits = ["Diamonds", "Clubs", "Hearts", "Spades"];
+    if (list.length === 0) {
+      console.log(list);
+      return null;
+    }
+    if (list[0].rank === list[1].rank) {
```

Comment:
Could list[0] or list[1] possibly be undefined? If so, you might face a bug here